packages/itwin/grouping-mapping-widget/src/components/Properties/CalculatedProperties/CalculatedPropertyTable.tsx

```diff
@@ -107,15 +107,18 @@ export const CalculatedPropertyTable = ({
   const { mutateAsync: deleteProperty } = useMutation({
     mutationFn: async (propertyId: string) => {
       const accessToken = await getAccessToken();
-      await mappingClient.deleteCalculatedProperty(
+      await propertiesClient.deleteProperty(
         accessToken,
-        iModelId,
         mappingId,
         groupId,
         propertyId,
       );
     },
-    onSuccess: async () => queryClient.invalidateQueries({ queryKey: ["calculatedProperties", iModelId, mappingId, groupId] }),
+    onSuccess: async () => {
+      await queryClient.invalidateQueries({ queryKey: ["calculatedProperties", iModelId, mappingId, groupId] });
```

Comment:
I think the `properties` should not be cached separately. Query and store them as a single action and only split them into separate lists when rendering.

Reply:
Done

Reply:
Key should be "properties" instead.

Reply:
`queryKey` changed to "properties".